client/app/cloud/project/compute/infrastructure/privateNetwork/addVRack/cloud-project-compute-infrastructure-privateNetwork-addVRack.controller.js

```diff
@@ -0,0 +1,82 @@
+class AddVRackCtrl {
+    constructor($window, $uibModalInstance, $stateParams, params, OvhApiVrack) {
+        this.projectId = $stateParams.projectId;
+        this.orderUrl = params.orderUrl;
+        this.modal = $uibModalInstance;
+        this.$window = $window;
+        this.OvhApiVrack = OvhApiVrack;
+        //get vRacks for current user, shown in left side bar
+        this.vRacks = params.vRacks;
+        this.loaders = {
+            activate: false
+        };
+        //activate options ENUM
+        this.activateOptions = _.indexBy(["EXISTING", "NEW"]);
+        //selected activate option
+        this.selectedActivateOption = this.activateOptions.EXISTING;
+        //selected vRack
+        this.selectedVrack = null;
+        if(this.vRacks && _.isArray(this.vRacks) ) {
+            //make first vRack seected on UI drop downlist
```

Comment:
I don,t think we want the first vrack to be selected by default.   The vrack dropdown should behave like what was made before.  See : https://www.ovh.com/manager/cloud/#/network/iplb/ip-213.32.5.68/frontends/add

Reply:
fixed